domain/src/main/java/org/oppia/android/domain/oppialogger/loguploader/LogUploadWorker.kt

```diff
@@ -47,6 +52,7 @@ class LogUploadWorker private constructor(
       when (inputData.getStringFromData(WORKER_CASE_KEY)) {
         EVENT_WORKER -> uploadEvents()
         EXCEPTION_WORKER -> uploadExceptions()
+        PERFORMANCE_METRICS_WORKER -> uploadPerformanceMetrics()
```

Comment:
Any tests to verify that this is working as expected?

Reply:
added.